IndomitableIbises/Bangazon-API-Sprint-1
# changed HTTP request permissions for deletion to comply with tickets

## Description of Proposed Changes

- added HTTP requests to view

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout SI_http
```
nuke your DB with django_data.sh API faker_factory
server up and make sure you can no longer delete employees, customers or departments



## Impacted Areas in Application

views.py

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
